portality/forms/application_forms.py

```diff
@@ -175,7 +175,7 @@ class FieldDefinitions:
         },
         "widgets": [
             "trim_whitespace",
-            "full_contents"
+            {"full_contents" : {"empty_disabled" : "[The journal has no alternative title]"}}
```

Comment:
@sssoz just drawing your attention to this, so you can tune the text as you need

Reply:
Looks good! Thanks for this.